docs/usage.md

```diff
@@ -25,7 +25,8 @@ Table of Contents
   * [Programmer checking new bugs in the code after local edit (and compare it to a central database)](#compare)
   * [Setting up user authentication](#authentication)
 * [Updating CodeChecker to new version](#upgrade)
-
+* [Unique Report Identifier](#unique-report-indentifier)
+* [Listing and Counting Reports](#listing-reports)
```

Comment:
Could you please add links to subsection of this section:
- [How reports are counted?](#how-reports-are-counted)
- [Report Uniqueing](#report-uniqueing)
- [How diffs between runs are calculated?](#how-diffs-between-runs-are-calculated)